src/io/iomanager.cpp

```diff
@@ -43,21 +45,27 @@ static void get_protobuf_msg_of_universe(missiondmx::fish::ipcmessages::Universe
 			// not finished
 			universe_to_edit->set_physical_location(1);
 			break;
-		case dmxfish::dmx::universe_type::ARTNET:
-			{
-				auto universe_inner = universe_to_edit->mutable_remote_location();
-				universe_inner->set_ip_address("Dummy data - we cant get real one right now");
-				universe_inner->set_port(6454);
-				universe_inner->set_universe_on_device(1);
-				break;
-			}
-		case dmxfish::dmx::universe_type::sACN:
-			// not finished, not supported in Protobuf
-			break;
+		case dmxfish::dmx::universe_type::ARTNET:{
+            auto universe_inner = universe_to_edit->mutable_remote_location();
+            universe_inner->set_ip_address("Dummy data - we cant get real one right now");
+            universe_inner->set_port(6454);
+            universe_inner->set_universe_on_device(1);
+            break;
+        }
+        case dmxfish::dmx::universe_type::sACN:
+            // not finished, not supported in Protobuf
+            break;
+        case dmxfish::dmx::universe_type::FTDI: {
+            auto universe_inner = universe_to_edit->mutable_ftdi_dongle();
+            universe_inner->set_product_id(1);
+            universe_inner->set_vendor_id(1);
+            universe_inner->set_device_name("Hopefully Enttec USB DMX Pro");
+            universe_inner->set_serial("Dummy Serial");
```

Comment:
Soll ich getter in ftdi_universe fuer die Verbindungsdaten anlegen?

Reply:
Gerne :)
Aber gibt es das schon für Artnet?

Reply:
Ich habe das einfach mal mit 960572b implementiert.